sakura_core/window/CMainStatusBar.h

```diff
@@ -56,6 +56,8 @@ class CMainStatusBar : public CDocListenerEx{
 
 	//設定
 	bool SetStatusText(int nIndex, int nOption, const WCHAR* pszText, size_t textLen = SIZE_MAX);
+	void ShowProgressBar() const;	// プログレスバーを表示する
```

Comment:
書きっぷりの提案（あまり重要ではない）
```suggestion
	void ShowProgressBar() const;
```
👆見たらわかるからコメント要らなくね？とか。
```suggestion
	void ShowProgressBar() const;	//!< プログレスバーを表示する
```
👆せっかくコメント付けるのだから、doxygenコメントにしよう、とか。